ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/qos_profiles.schema.yml

```diff
@@ -81,6 +81,60 @@ keys:
                 type: str
                 description: |
                   Text comment added to queue.
+              random_detect:
+                type: dict
+                keys:
+                  ecn:
+                    description: Explicit Congestion Notification
+                    type: dict
+                    keys:
+                      count:
+                        type: bool
+                        description: |
+                          Enable counter for random-detect ECNs
+                      threshold:
+                        type: dict
+                        keys:
+                          units:
+                            required: true
+                            type: str
+                            valid_values: ["segments", "bytes", "kbytes", "mbytes"]
+                            description: |
+                              Units to be used for the threshold values.
+                              This should be one of segments, byte, kbytes, mbytes.
+                          min:
+                            type: int
+                            required: true
+                            min: 1
+                            convert_types:
+                              - str
+                            description: |
+                              Random-detect ECN minimum-threshold
+                          max:
+                            type: int
+                            required: true
+                            min: 1
+                            convert_types:
+                              - str
+                            description: |
+                              Random-detect ECN maximum-threshold
+                          max_probability:
+                            type: int
+                            min: 1
+                            max: 100
+                            convert_types:
+                              - str
+                            description: |
+                              Random-detect ECN maximum mark probability.
+                          weight:
+                            type: int
+                            min: 1
```

Comment:
the min for the weight could be between 0 and 15, see cli:

ld002(config-s-test-qos-profile-NET-INFRA-txq-4)# random-detect ecn minimum-threshold 256 kbytes maximum-threshold 512 kbytes weight ?
  <0-15>  Value of ECN weight

Reply:
Agreed - will be channging all references.